discovery-provider/src/tasks/index_user_bank.py

```diff
@@ -653,21 +655,32 @@ def parse_create_token_data(data: str) -> CreateTokenAccount:
 def process_user_bank_tx_details(
     session: Session,
     redis: Redis,
-    tx_info: ConfirmedTransaction,
+    tx_info: GetTransactionResp,
     tx_sig,
     timestamp,
     challenge_event_bus: ChallengeEventBus,
 ):
-    result: TransactionInfoResult = tx_info["result"]
-    meta = result["meta"]
-    error = meta["err"]
+    result = tx_info.value
+    if not result:
+        logger.error("index_user_bank.py | No result")
+        return
+    meta = result.transaction.meta
+    if not meta:
+        logger.error("index_user_bank.py | No result meta")
+        return
+    error = meta.err
     if error:
         logger.info(
             f"index_user_bank.py | Skipping error transaction from chain {tx_info}"
         )
         return
-    account_keys = tx_info["result"]["transaction"]["message"]["accountKeys"]
-    tx_message = result["transaction"]["message"]
+    account_keys = list(
+        map(
+            lambda x: str(x),
+            cast(Transaction, result.transaction.transaction).message.account_keys,
```

Comment:
nit: can we avoid the `cast` somehow ?

Reply:
<img width="1064" alt="Screenshot 2023-08-05 at 2 07 20 PM" src="https://github.com/AudiusProject/audius-protocol/assets/2731362/775a548f-c592-4db2-ba3a-ae865687f3f7">

I think this is a shortcoming of solders. It could be some other things that don't have a message. This cast is bad. It is a lie.

This is the annoying fix

```
    transaction = result.transaction.transaction
    if not hasattr(transaction, 'message'):
        logger.info(f"index_user_bank.py | No transaction message found {transaction}")
        return

    account_keys = list(map(lambda x: str(x), transaction.message.account_keys))
```